source/templates.tex

```diff
@@ -1375,11 +1377,7 @@
 \tcode{float},
 and list of pointers to
 \tcode{Shape}s.
```

Comment:
Done, now it looks like this:

![image](https://cloud.githubusercontent.com/assets/6378233/21205536/a3cc3894-c254-11e6-91c8-300ee0d6e91f.png)